README.md

```diff
@@ -11,7 +11,7 @@ This cookbook is used to install and configure Gluster on both servers and clien
 
 Platforms
 ---------
-This cookbook has been tested on Ubuntu 12.04/14.04 and CentOS 6.5.
+This cookbook has been tested on Ubuntu 12.04/14.04, CentOS 6.5 and CentOS 7.1
```

Comment:
Sure, but the kitchen.yml doesn't really test functionality properly anyway. I am trying to write a better one at the moment